docs/administration/managing-users-and-teams/creating-teams-for-a-user-with-mixed-environment-privileges.md

```diff
@@ -4,39 +4,46 @@ description: This guide describes how to create a teams for a user with mixed en
 position: 10
 ---
 
-A common scenario some users may face, is the desire to provide full access to one environment, but only read access to the next stage. For example the developers might be able to fully manage deployments to the development and staging environment, but only view the production deployments.
+A common scenario some users may face, is the desire to provide full access to one environment, but only read access to the next stage. For example the developers might be able to fully manage deployments to the development and staging environments, but only view the production deployments. Let's build this out with a team and some user roles. 
 
-Given that a set of roles can be combined in a team and tied to a specific environment, this scenario indicates that we will need two different teams. One with the full deployment access but scoped to dev and staging, and another with just read roles scoped to production.
+In versions prior to **2018.11**, this scenario would have required two different teams. One with the full deployment access but scoped to dev and staging, and another with just read roles scoped to production.
```

Comment:
A minor point, but I think we can drop comments about how it was achieved before 2018.11 and just talk about how to do it now. (references to previous ways of doing things provide a nice comparison for existing users, but can be confusing for new users not familiar with the old way). Users working with older versions of Octopus can access the legacy docs from the version selector at the top of the page.

Reply:
Good point - I'll drop such references - thanks.